src/cssUsage.js

```diff
@@ -605,69 +605,82 @@ void function() { try {
+
+                        // Remove group contents (...), {...} and [...]
+                        value = value.replace(/[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]*)[)])*[)])*[)])*[)])*[)]/g, " <parentheses-block> ");
+                        value = value.replace(/[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]*)[)])*[)])*[)])*[)])*[)]/g, " <parentheses-block> ");
+                        value = value.replace(/[{](?:[^{}]+|[{](?:[^{}]+|[{](?:[^{}]+|[{](?:[^{}]+|[{](?:[^{}]*)[}])*[}])*[}])*[}])*[}]/g, " <curly-brackets-block> ");
+                        value = value.replace(/[{](?:[^{}]+|[{](?:[^{}]+|[{](?:[^{}]+|[{](?:[^{}]+|[{](?:[^{}]*)[}])*[}])*[}])*[}])*[}]/g, " <curly-brackets-block> ");
+                        value = value.replace(/[\[](?:[^\[\]]+|[\[](?:[^\[\]]+|[\[](?:[^\[\]]+|[\[](?:[^\[\]]+|[\[](?:[^\[\]]*)[\]])*[\]])*[\]])*[\]])*[\]]/g, " <square-brackets-block> ");
+                        value = value.replace(/[\[](?:[^\[\]]+|[\[](?:[^\[\]]+|[\[](?:[^\[\]]+|[\[](?:[^\[\]]+|[\[](?:[^\[\]]*)[\]])*[\]])*[\]])*[\]])*[\]]/g, " <square-brackets-block> ");
+
+                        break;
+
+                    default:
+
+                        // Replace strings by dummies
+                        value = value.replace(/"([^"\\]|\\[^"\\]|\\\\|\\")*"/g,' <string> ')
+                            .replace(/'([^'\\]|\\[^'\\]|\\\\|\\')*'/g,' <string> ');
+
+                        // Replace url(...) functions by dummies
+                        if (value.indexOf("(") != -1) {
+                            value = value.replace(/([a-z]?)[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]*)[)])*[)])*[)])*[)])*[)]/g, "$1() ");
+                            value = value.replace(/([a-z]?)[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]*)[)])*[)])*[)])*[)])*[)]/g, "$1() ");
+                        }
+
+                }
+            }
+            else {
+                switch (propertyName) {
+                    // If a URL value is given for the cursor property, then we want to remove all the speech marks put in around the url links
+                    // to prevent inconsistencies in results (in some running cases they are put in, others not) and split on the fallback values supplied.
+                    case 'cursor':
+
+                        // Remove various quotes - Crawler has some issue relating to speech marks, where sometimes they are put around url links (local test page run) and sometimes not (crawler run).
+                        if (value.indexOf("'") != -1 || value.indexOf("‘") != -1 || value.indexOf('"')) {
+                            value = value.replace(/('|‘|’|")/g, "");
+                        }
+
+                        // Divide at commas to separate cursor url value and supplied fallback values.
+                        value = value.split(/\s*,\s*/g);
+                        return value;

```

Comment:
I don't like that we have an "else" statement that returns a value when the "if" branch doesn't, it's easy to miss when you are ready the code. I'd much rather this be its own statement even if this means repeating the condition.

Reply:
We're only returning a value for the cursor property to miss out the bits after, like we do in font-family for example. I am keeping to the convention I believe.